.circleci/config.yml

```diff
@@ -457,3 +459,20 @@ workflows:
               ignore: /tests\/.*/
             tags:
               only: /.*/
+
+# Circle API v2 build trigger parameters
+# Ref: https://circleci.com/docs/2.0/pipeline-variables/
+# Ref: https://circleci.com/docs/2.0/reusing-config/#parameter-syntax
+parameters:
```

Comment:
These correspond to https://github.com/Automattic/wp-desktop-gh-bridge/pull/13/files#diff-168726dbe96b3ce427e7fedce31bb0bcR254-R258, which makes sense to me!